site/plugins.json

```diff
@@ -374,7 +374,8 @@
     "version": "1.0.0",
     "compatibility": [
       {
-        "version": "0.1.1"
+        "version": "0.1.1",
+        "migrationGuide": "https://gitlab.com/html-validate/html-validate/-/blob/master/CHANGELOG.md#-breaking-changes"
```

Comment:
We could also link to https://github.com/oliverroick/netlify-plugin-html-validate/releases/tag/v1.0.0
Not sure users will understand the connection between the plugin and `html-validate` due to different versioning

Reply:
Yes, that's better! 
Done in https://github.com/netlify/plugins/pull/318